### PR TITLE
Update QNAP integration

### PR DESCRIPTION
**Pull request by cyr-ius**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [#50302](https://github.com/home-assistant/core/pull/50302)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: source/_integrations/qnap.markdown [hunk 1]
```diff
@@ -7,116 +7,28 @@ ha_release: 0.38
 ha_iot_class: Local Polling
 ha_codeowners:
   - '@colinodell'
+  - '@cyr-ius'
 ha_domain: qnap
 ha_platforms:
   - sensor
 ---
 
 This `qnap` sensor allows getting various statistics from your [QNAP NAS](https://www.qnap.com/en-us/).
```
Comment (frenck) [inline source/_integrations/qnap.markdown:16]:
```suggestion
The QNAP integration allows getting various statistics from your [QNAP NAS](https://www.qnap.com/en-us/).
```

File: source/_integrations/qnap.markdown [hunk 2]
```diff
@@ -7,116 +7,28 @@ ha_release: 0.38
 ha_iot_class: Local Polling
 ha_codeowners:
   - '@colinodell'
+  - '@cyr-ius'
 ha_domain: qnap
```
Comment (frenck) [inline source/_integrations/qnap.markdown:11]: - Missing `ha_config_flow`.

Does it have discovery now as well? In that case, that needs to be added as well.

File: source/_integrations/qnap.markdown [hunk 3]
```diff
@@ -7,116 +7,28 @@ ha_release: 0.38
...
-      description: Displays the used space of the volume as a percentage (creates a new entry for each volume).
-{% endconfiguration %}
+Sensors:
+- system: status, system temperature
+- cpu : usage, temperature
+- memory : usage, free, used, total
+- volume : usage, used
+- folders : usage, used
+- drives: smart status, usage
+- networks uplink : bandwidth, status
```
Comment (frenck) [inline source/_integrations/qnap.markdown:31]:
```suggestion
Sensors:

- System: status, system temperature
- CPU: usage, temperature
- Memory: usage, free, used, total
- Volume: usage, used
- Folders: usage, used
- Drives: smart status, usage
- Network: bandwidth, status
```

File: source/_integrations/qnap.markdown [hunk 4]
```diff
@@ -7,116 +7,28 @@ ha_release: 0.38
...
 
 ### SSL and Self-signed certificates
```
Comment (frenck) [inline source/_integrations/qnap.markdown:33]: This section needs adjustment as well, as it references to YAML configuration that has been removed.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**